src/components/cards/remote-cards/RemoteCardProvider.tsx

```diff
@@ -0,0 +1,132 @@
+import { noop, pick } from 'lodash';
+import React, { PropsWithChildren, createContext, useState } from 'react';
+import { Card } from '@/graphql/__generated__/arc';
+import Routes from '@/navigation/routesNames';
+import {
+  CardCollectionResult,
+  useCardCollectionQuery,
+} from '@/resources/cards/cardCollectionQuery';
+import * as ls from '@/storage';
+import { REMOTE_CARDS, useExperimentalFlag } from '@/config';
+
+const TRIMMED_CARD_KEYS = [
+  'cardKey',
+  'placement',
+  'index',
+  'backgroundColor',
+  'padding',
+  'imageRadius',
+  'subtitle',
+  'subtitleColor',
+  'title',
+  'titleColor',
+  'description',
+  'descriptionColor',
+  'items',
+  'primaryButton',
+] as const;
+
+export type TrimmedCard = Pick<Card, typeof TRIMMED_CARD_KEYS[number]> & {
+  sys: Pick<Card['sys'], 'id'>;
+  imageCollection: {
+    items: {
+      url: string;
+    }[];
+  };
+};
+
+type CardProviderProps = {
+  initialState?: Record<keyof TrimmedCard['cardKey'], TrimmedCard>;
+};
+
+type CardContextProps = {
+  cards: Record<keyof TrimmedCard['cardKey'], TrimmedCard>;
+  setCards: React.Dispatch<
+    React.SetStateAction<Record<keyof TrimmedCard['cardKey'], TrimmedCard>>
+  >;
+  dismissCard: (cardKey: keyof TrimmedCard['sys']['id']) => void;
+  getCardsForPlacement: (placement: string) => TrimmedCard[];
+};
+
+export const RemoteCardContext = createContext<CardContextProps>({
+  cards: {},
+  setCards: noop,
+  dismissCard: noop,
+  getCardsForPlacement: () => [],
+});
+
+export const RemoteCardProvider: React.FC<
+  PropsWithChildren<CardProviderProps>
+> = ({ children }) => {
+  const [cards, setCards] = useState<
+    Record<keyof TrimmedCard['cardKey'], TrimmedCard>
+  >({});
+  const enabled = useExperimentalFlag(REMOTE_CARDS);
+
+  useCardCollectionQuery(
+    {},
+    {
+      enabled,
+      refetchInterval: 5_000,
```

Comment:
```suggestion
      refetchInterval: 60_000,
```
change interval after testing